docs/getting_started/dashboard_copilot.md

```
@@ -280,16 +280,16 @@ The aggregations in these charts will be mined for metric definitions. By defaul
 
 **6. Pick a thread-pool size**
 
-The build phase issues parallel LLM calls. Increase the pool size to speed things up if your provider quota allows.
+The build phase issues parallel LLM calls. Increase the pool size to speed things up if your provider quota allows. The default is 3.
 
 ```text
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)